modules/nw-ptp-installing-operator.adoc

```diff
@@ -31,6 +31,7 @@ metadata:
   name: openshift-ptp
   labels:
     openshift.io/run-level: "1"
```

Comment:
This line should be removed, and add  'name: openshift-ptp'. The whole yaml file is as below:

apiVersion: v1
kind: Namespace
metadata:
  name: openshift-ptp
  labels:
    name: openshift-ptp
    openshift.io/cluster-monitoring: "true"

Reply:
@huiran0826, updated!

Reply:
Thanks! Others LGTM